docs/manual/treefile.md

```diff
@@ -224,6 +224,12 @@ It supports the following parameters:
    also be an array of strings.  Including the same file multiple times
    is an error.
 
+ * `arch-include`: object (`Map<String,include>`), optional: Each member of this
+   object should be the name of an architecture, and the `include` value
+   functions the same as the `include` key above - it can be either
+   a single string, or an array of strings - and it has the same semantics.
+   `arch-include` is processed after `include`.
```

Comment:
To be more explicit here, how about: "Entries which match the `$basearch` of the target platform are processed after the regular `include`" ?